docs/Glossary/index.md

```diff
@@ -0,0 +1,136 @@
+# Glossary
+
+This glossary provides definitions of commonly used terms and phrases, organized by application or function.
+
+---
+
+## Media Management Applications
+
+**Radarr**, **Sonarr**, and **Lidarr** are applications designed to manage your media libraries:
+
+- **Radarr**: For movies
+- **Sonarr**: For TV shows
+- **Lidarr**: For music
+
+### Custom Format
+
+A feature that matches specific aspects of a release based on information extracted from its name. Users choose the matching criteria, and Custom Formats can be assigned scores on a per-[Quality Profile](#quality-profile) basis.
+
+### Quality
+
+Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.
+
+### Quality Definitions
+
+Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.
+
+### Quality Profile
+
+A defined group of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.
+
+---
+
+## Torrents
+
+### Cross-Seed
+
+The act of [seeding](#seeder) the same torrent across multiple [trackers](#tracker). It also refers to a software application of the same name that automates this process.
+
+### Deluge
+
+A lightweight, free, open-source, cross-platform BitTorrent client.
+
+### qBit Manage
+
+A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes, such as time and tracker.
```

Comment:
```suggestion
A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes (such as time and tracker.)

Reply:
Preferred without the brackets, or indeed my previous version with a comma.